Added theming for TalkingUI

Krzmbrzl opened the pull request:
These styles will be used in the new TalkingUI feature.

davidebeatrici commented on Dark.qss:
```diff
@@ -1043,3 +1043,24 @@ AboutDialog QTextEdit {
+TalkingUI [selected=true] {
+  background-color: #3e4f5e;
+  border: 1px solid #3e4f5e;
+}
```
Keep newline at the bottom.

Krzmbrzl replied:
This is actually how the preprocessor program thingy generated it... But I can add it manually :)

davidebeatrici commented on Lite.qss:
```diff
@@ -1043,3 +1043,24 @@ AboutDialog QTextEdit {
+TalkingUI [selected=true] {
+  background-color: #dcedf5;
+  border: 1px solid #97b5c6;
+}
```
Keep newline at the bottom.

davidebeatrici commented on OSX Dark.qss:
```diff
@@ -1064,4 +1085,4 @@ QToolButton:on {
 QToolButton:on:pressed {
   background-color: transparent;
-}
+}
```
Keep newline at the bottom.

davidebeatrici commented on OSX Lite.qss:
```diff
@@ -1064,4 +1085,4 @@ QToolButton:on {
 QToolButton:on:pressed {
   background-color: transparent;
-}
+}
```
Keep newline at the bottom.

Krzmbrzl commented:
@davidebeatrici I added the missing newlines.

Additionally I added a small note to the README as this was something I had a bit trouble understanding. Maybe it'll help someone in the future. I figured that I'd simply include it in this PR for the sake of simplicity :)
